src/app/styles/components/_variables.scss

```diff
@@ -6,3 +6,6 @@ $headerbar-background: #466BC1;
 $button-hover-color: #3859A5;
 $button-padding: 12px;
 $logout-button-color: #FFFFFF;
+$infokit-z-index: 2000;
+$infokit-popup-background: rgba(0, 0, 0, 0.7);
+$infokit-font-color: #FFFFFF;
```

Comment:
similar to what i've said [here](https://github.com/systers/PC-Prep-Kit/pull/25/files/4eb1d54fef7291b612ddd8a9da5dc1c504b82bfc#r128696230)
you are defeating the purpose of scss variables
please coordinate with Raxerz & come up with an agreement on how variables in scss should be named & used